src/test/java/api/requests/scenarios/RequestQueueTests.java

```diff
@@ -189,9 +189,9 @@ void canFetchTheRequestQueueForAnItem() {
 
       JsonObject item = request.getJsonObject("item");
 
-      assertThat(String.format("item summary has a holdings record ID: %s",
+      assertThat(String.format("item summary has not a holdings record ID: %s",
```

Comment:
Personally, I think this assert message looks a bit weird, I don't think we need it at all...